DoLah/Commands/Command.h

```diff
@@ -117,6 +117,22 @@ namespace DoLah {
         std::vector<AbstractTask*> ** resultVector;
 	};
 
+    class SearchDateTaskCommand : public AbstractCommand {
+    public:
+        SearchDateTaskCommand();
+        SearchDateTaskCommand(std::tm, std::tm);
+        SearchDateTaskCommand(std::tm, std::tm, std::vector<AbstractTask*> *);
+        ~SearchDateTaskCommand();
+
+        void execute();
+        void revert();
```

Comment:
SearchTaskCommand has invertSearch which uses revert() function, so I added it for SearchDateTaskCommand as well just in case. The issue #204 is raised to clarify this.